<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>secondWindowResearcher</class>
 <widget class="QDialog" name="secondWindowResearcher">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>400</width>
    <height>300</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Researcher Menu</string>
  </property>
  <widget class="QLabel" name="label">
   <property name="geometry">
    <rect>
     <x>160</x>
     <y>10</y>
     <width>81</width>
     <height>16</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>MingLiU</family>
    </font>
   </property>
   <property name="text">
    <string>Researcher</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_2">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>40</y>
     <width>161</width>
     <height>16</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>PingFang SC</family>
    </font>
   </property>
   <property name="text">
    <string>Select Type Of Model:</string>
   </property>
  </widget>
  <widget class="QCheckBox" name="checkBox_3">
   <property name="geometry">
    <rect>
     <x>60</x>
     <y>100</y>
     <width>211</width>
     <height>21</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>PingFang SC</family>
    </font>
   </property>
   <property name="text">
    <string>View Model By A Researcher</string>
   </property>
  </widget>
  <widget class="QCheckBox" name="checkBox_4">
   <property name="geometry">
    <rect>
     <x>60</x>
     <y>130</y>
     <width>201</width>
     <height>21</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>PingFang SC</family>
    </font>
   </property>
   <property name="text">
    <string>Create A New Model </string>
   </property>
  </widget>
  <widget class="QPushButton" name="pushButton">
   <property name="geometry">
    <rect>
     <x>270</x>
     <y>250</y>
     <width>114</width>
     <height>32</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>PingFang SC</family>
    </font>
   </property>
   <property name="text">
    <string>Continue</string>
   </property>
  </widget>
  <widget class="QCheckBox" name="checkBox_5">
   <property name="geometry">
    <rect>
     <x>60</x>
     <y>160</y>
     <width>201</width>
     <height>21</height>
    </rect>
   </property>
   <property name="text">
    <string>View Real TIme Raw Data</string>
   </property>
  </widget>
  <widget class="QCheckBox" name="checkBox_2">
   <property name="geometry">
    <rect>
     <x>60</x>
     <y>70</y>
     <width>281</width>
     <height>21</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>PingFang SC</family>
    </font>
   </property>
   <property name="text">
    <string>View Real Time Simulation Of Bee Hive</string>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
